test_scripts/Polices/build_options/032_ATF_P_TC_PTU_Restarting_Policy_Table_Exchange_HTTP.lua

```diff
@@ -74,156 +81,185 @@ local function DelayedExp(time)
-  firstTryTime = startPTUtime + timeout_after_x_seconds + seconds_between_retries[1]
-  return true
-end
+        syncMsgVersion =
+        {
+          majorVersion = 3,
+          minorVersion = 0
+        },
+        appName = "AppName",
+        isMediaApplication = true,
+        languageDesired = "EN-US",
+        hmiDisplayLanguageDesired = "EN-US",
+        appID = "1234567",
+        deviceInfo =
+        {
+          os = "Android",
+          carrier = "Megafon",
+          firmwareRev = "Name: Linux, Version: 3.4.0-perf",
+          osVersion = "4.4.2",
+          maxNumberRFCOMMPorts = 1
+        }
+      })
+    EXPECT_NOTIFICATION("OnSystemRequest")
+    :ValidIf(function(exp,data)
+        if(data.payload.requestType == "HTTP") then
 
-if exp.occurences == 2 and firstTryTime == os.time() then 
-  secondTryTime = timeout_after_x_seconds + firstTryTime + seconds_between_retries[2]
-  print ("first retry time: " .. os.time())
-  return true
-  elseif exp.occurences == 2 and firstTryTime ~= os.time() then 
-    print ("Wrong first retry time! Expected: " .. timeout_after_x_seconds + seconds_between_retries[1] .. " Actual: " .. os.time() - startPTUtime)
-    return false
-end
+          if exp.occurences == 2 then
+            startPTUtime = os.time()
+            firstTryTime = timeout[1]
+            return true
+          end
 
-if exp.occurences == 3 and secondTryTime == os.time() then 
-  thirdTryTime = timeout_after_x_seconds + secondTryTime + seconds_between_retries[3]
-  print ("second retry time: " .. os.time())
-  return true
-  elseif exp.occurences == 2 and secondTryTime ~= os.time() then 
-    print ("Wrong second retry time! Expected: " .. timeout_after_x_seconds + seconds_between_retries[2] .. " Actual: " .. os.time() - firstTryTime)
-    return false
-end
+          if exp.occurences == 3 and firstTryTime == os.time() then
+            secondTryTime = timeout[2]
+            print ("first retry time: " .. os.time())
+            return true
+          elseif exp.occurences == 2 and firstTryTime ~= os.time() then
+            print ("Wrong first retry time! Expected: " .. timeout[1] .. " Actual: " .. os.time() - startPTUtime)
+            return false
+          end
 
-if exp.occurences == 4 and thirdTryTime == os.time() then 
-  print ("third retry time: " .. os.time())
-  return true
-  elseif exp.occurences == 2 and thirdTryTime ~= os.time() then 
-    print ("Wrong third retry time! Expected: " .. timeout_after_x_seconds + seconds_between_retries[3] .. " Actual: " .. os.time() - secondTryTime)
-    return false
-end
-return false
+          if exp.occurences == 4 and secondTryTime == os.time() then
+            thirdTryTime = timeout[3]
+            print ("second retry time: " .. os.time())
+            return true
+          elseif exp.occurences == 3 and secondTryTime ~= os.time() then
+            print ("Wrong second retry time! Expected: " .. timeout[2] .. " Actual: " .. os.time() - firstTryTime)
```

Comment:
@istoimenova, For the 2nd and next retries actual is calculated incorrectly.
os.time() gives value in seconds starting from January 1, 1970 (ex.: 1493805155)
firstTryTime, secondTryTime etc. are represented in seconds
So actual result will be in seconds starting from January 1, 1970 (ex.: 1493805125)
But expected timeout[x] is in seconds